doc/news/changes/incompatibilities/20230721Munch

```diff
@@ -0,0 +1,7 @@
+Removed: The methods copy_to_mg, copy_from_mg,
+and interpolate_to_mg used of
+MGTransferGlobalCoarsening and MGTransferMatrixFree
+used to have spacedim as template argument. Since it
+is not used, it has been removed.
```

Comment:
```suggestion
used to have spacedim as template argument. Since it
is not used and the underlying functionality did not work,
it has been removed.
```